app/breakpoints.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  mobile:  '(max-width: 767px)',
```

Comment:
[Bootstrap defines media break points](https://github.com/twbs/bootstrap-sass/blob/master/templates/project/_bootstrap-variables.sass#L287), we should use those if required.